.gitignore

```diff
@@ -1,5 +1,7 @@
 /bin
 /out
+/build
```

Comment:
Is there any particular reason we need another build directory? It seems like the out directory is already used for building.

Reply:
I guess that can work too. I'm just used to using `build` 🙂 .

Reply:
Fixed.